bayesian/utils.py

```diff
@@ -19,9 +19,16 @@
 from f8a_worker.setup_celery import init_celery
 from .default_config import STACK_ANALYSIS_REQUEST_TIMEOUT
 from sqlalchemy.exc import SQLAlchemyError
+from f8a_utils.ingestion_utils import trigger_workerflow
 
 logger = logging.getLogger(__name__)
 
+_INGESTION_API_URL = "http://{host}:{port}/{endpoint}".format(
```

Comment:
You don't need this, its already defined in f8a_utils

Reply:
My bad ,forgot to remove this from last implementation.